src/styles/responsive.js

```diff
@@ -1,30 +1,113 @@
-const COLUMN_WIDTH =    104               // 1/2 Item Card width
+const COLUMN_WIDTH = 104                  // 1/2 Item Card width
 
-const SCREEN_SM_MIN =   768               // 6 columns rounded
-const SCREEN_MD_MIN =   COLUMN_WIDTH * 8  // 832
+const SCREEN_SM_MIN = 768                 // 6 columns rounded
+const SCREEN_MD_MIN = COLUMN_WIDTH * 8    // 832
 const SCREEN_MDLG_MIN = COLUMN_WIDTH * 10 // 1040
-const SCREEN_LG_MIN =   COLUMN_WIDTH * 12 // 1248
-const SCREEN_XL_MIN =   COLUMN_WIDTH * 14 // 1456 (max width)
+const SCREEN_LG_MIN = COLUMN_WIDTH * 12   // 1248
+const SCREEN_XL_MIN = COLUMN_WIDTH * 14   // 1456 (max width)
 
 const SCREEN_XS_MAX = SCREEN_SM_MIN - 1   // 767
 const SCREEN_SM_MAX = SCREEN_MD_MIN - 1   // 831
 const SCREEN_MD_MAX = SCREEN_MDLG_MIN - 1 // 1039
 const SCREEN_MDLG_MAX = SCREEN_LG_MIN - 1 // 1247
 const SCREEN_LG_MAX = SCREEN_XL_MIN - 1   // 1455
 
+export const breakpoints = {
+  xs: {
+    min: 0,
+    max: SCREEN_XS_MAX
+  },
+  sm: {
+    min: SCREEN_SM_MIN,
+    max: SCREEN_SM_MAX
+  },
+  md: {
+    min: SCREEN_MD_MIN,
+    max: SCREEN_MD_MAX
+  },
+  mdLg: {
+    min: SCREEN_MDLG_MIN,
+    max: SCREEN_MDLG_MAX
+  },
+  lg: {
+    min: SCREEN_LG_MIN,
+    max: SCREEN_LG_MAX
+  },
+  xl: {
+    min: SCREEN_XL_MIN
+  }
+}
+
+const assertValidSizes = (...sizes) => {
+  sizes.forEach(size => {
+    if (breakpoints[size] === undefined) {
+      throw new Error(`Screen size(s) ${sizes.join(', ')} not supported. Must be one of ${Object.keys(breakpoints).join(', ')}`) // eslint-disable-line
```

Comment:
nice! We should probably do more of this kind of thing to help out the users of this library.

Looks like I need to edit the eslint rule for this to disregard strings

Reply:
Yeah, I figured this was better than returning a broken media query. One thing I was curious about was whether I could/should make this dev-only 🤔